iceoryx_utils/test/moduletests/test_message_queue.cpp

```diff
@@ -24,7 +24,11 @@ using namespace ::testing;
 using namespace iox;
 using namespace iox::posix;
 
-using IpcChannel = MessageQueue;
+#if defined(__APPLE__)
+using IpcChannelType = UnixDomainSocket;
```

Comment:
why not just include the `iceoryx_posh_types.hpp`?
I think you misunderstood my previous request. The idea was to include `iceoryx_posh_types.hpp` and just add a `using IpcChannel = iox::IpcChannelType;` to get rid of the platform dependent code.

Reply:
Yes I did this intentionally. If I would use `iceoryx_posh_types.hpp` from `iceoryx_posh` the `iceoryx_utils` would have a dependency to posh which I do not want.

Reply:
oh, my bad. I feel kind of stupid now